sparkctl/build.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+#
+# Copyright 2019 The Kubernetes Authors.
```

Comment:
Isn't the header supposed to be the Google one used in other operator code files? "Kubernetes Authors" don't seem right here.

Reply:
I just used what the other scripts have, there are two kinds right now.

Reply:
No worries, I will fix them later.